org.uqbar.project.wollok.lib/src/wollok/lib.wlk

```diff
@@ -180,7 +180,7 @@ object game {
 	 * Adds an object to the board for drawing it.
 	 * That object should known a position.
 	 */
-	method addVisual(element) native
+	method addVisual(visual) native
```

Comment:
Perhaps can be 

method addVisual(domainObject) native  ??
method addVisual(visualObject) native  ??
method addVisual(objectToView) native  ??

I'm not sure ....